### ISSUE TITLE
Incorrect spell checking in Markdown fenced code blocks

### ISSUE THREAD

**Issue by borekb** [bartosz-antosik/vscode-spellright#151]
It seems that some combination of `<` and `>` characters cause incorrect spellchecking in subsequent fenced code blocks in Markdown. For example:

![image](https://user-images.githubusercontent.com/101152/39523984-c942a9a0-4e17-11e8-9b2c-d5d660975caa.png)

Here is a textual representation:

~~~
To reproduce, there needs to be `<` in a code block, e.g.:

```
x < 2
```

And then some `>` anywhere in the following text, e.g., here:

Go to Settings > Core.

All subsequent code blocks are suddenly subject to spell checking:

```
$ sdsd
```

```
$ adsfsddf
```
~~~

**Comment by bartosz-antosik** [comment]
Should be resolved in version 2.3.3. Cheers.

**Comment by borekb** [comment]
Awesome, thank you for a quick fix!
